server/scan/device_handling.py

```
@@ -1,764 +1,816 @@
+
+    Args:
+        vendor: Device vendor name.
+        mac: Device MAC address.
+        ip: Device IP address.
+        name: Device name.
+        default: Default icon to return if no match is found.
+
+    Returns:
+        str: Base64-encoded FontAwesome icon HTML string.
+    """
+    warnings.warn("guess_icon is deprecated; use guess_device_attributes instead", DeprecationWarning)
```

Ruff (0.11.9)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)